rdr_service/services/ancillary_studies/nph_incident.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from rdr_service import clock, config
+from rdr_service.services.slack_utils import SlackMessageHandler
+
+from rdr_service.model.study_nph import Incident as NphIncident
+from rdr_service.dao.study_nph_dao import NphIncidentDao
+
+
+_logger = logging.getLogger("rdr_logger")
+
+
+def get_slack_message_handler() -> SlackMessageHandler:
+    slack_config = config.getSettingJson(config.NPH_SLACK_WEBHOOKS, {})
+    if slack_config is None:
+        _logger.error("'slack_config' for 'NPH_SLACK_WEBHOOKS' is empty")
```

Comment:
We can just use `logging` for this, no need to change this to logger since these log output will live on the cloud.

Reply:
Okay